### PR TITLE
docs: improve external-api documentation

### PR DESCRIPTION
**Pull request by magalimorin18**
DEV-11279

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-11279 Improve swagger with more explanation on how to generate the PostDeploymentCallData](https://app.clickup.com/t/2645698/DEV-11279)

File: docs/tools/relayer-developer.md [hunk 1]
```diff
@@ -28,18 +28,132 @@ Please fill out [this form](https://forms.gle/rhWA25m3jjuPNPva9) to request acce
...
+// structure of the VerifiableURI
+0x0000 + 6f357c6a +       0020 +                    820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361 + 696670733a2f2f516d597231564a4c776572673670456f73636468564775676f3339706136727963455a4c6a7452504466573834554178
+  ^      ^                ^                         ^                                                                  ^
+  0000   keccak256(utf8)  verificationDatalength    verificationData                                                   encoded URL
+
+// example value
+0x00006f357c6a0020820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361696670733a2f2f516d597231564a4c776572673670456f73636468564775676f3339706136727963455a4c6a7452504466573834554178
+```
+
+ℹ️ More info on [VerifiableURI](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#verifiableuri)
```
Comment (CJ42) [inline docs/tools/relayer-developer.md:89]:
```suggestion
ℹ️ More info on [`VerifiableURI`](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#verifiableuri)
```

File: docs/tools/relayer-developer.md [hunk 2]
```diff
@@ -28,18 +28,132 @@ Please fill out [this form](https://forms.gle/rhWA25m3jjuPNPva9) to request acce
...
+
+To be able to deploy a UP on the same address across different chains.
+
+**How to generate each parameter?**
+
+```javascript title="salt: A 32 bytes salt used to compute the deployment address of the contract."
+const salt = '0x' + crypto.randomBytes(32).toString('hex');
+```
+
+```javascript title="postDeploymentCallData: Call data which will be executed on the ERC725Account contract after deployment. Should contain the encoded setData transaction to set required permission setting and LSP3 Profile Data"
```
Comment (CJ42) [inline docs/tools/relayer-developer.md:106]: Is it not setDataBatch?

```suggestion
```javascript title="postDeploymentCallData: Calldata which will be executed on the ERC725Account contract after deployment. Should contain the encoded setDataBatch transaction to set the initial permissions and LSP3 Profile Data"
```

File: docs/tools/relayer-developer.md [hunk 3]
```diff
@@ -28,18 +28,132 @@ Please fill out [this form](https://forms.gle/rhWA25m3jjuPNPva9) to request acce
...
+</details>
+
+Please refer to the [Swagger](https://relayer-api.testnet.lukso.network/docs#/External%20Api%20Endpoints/UniversalProfileController_deployUniversalProfile) docs for more information.
 
 ### Register Users and Universal Profiles
 
 If your project has already created Universal Profiles (UP) for your users, you can register these profiles to our relayer so they can benefit from the gasless transactions experience provided by the LUKSO relayer.
 
-### How it works
+#### How it works
```
Comment (CJ42) [inline docs/tools/relayer-developer.md:156]: This will not show this section in the sidebar with 4 

```suggestion
### How it works
```

**Review by CJ42 — CHANGES_REQUESTED** [review]

File: docs/tools/relayer-developer.md [hunk 4]
```diff
@@ -28,18 +28,132 @@ Please fill out [this form](https://forms.gle/rhWA25m3jjuPNPva9) to request acce
...
-- or a `salt` and `postDeploymentCallData`
+- OPTION 1: a list of controller addresses (`lsp6ControllerAddress`) and metadata (`lsp3Profile`)
+- OPTION 2: a `salt` and `postDeploymentCallData`
 
-Please refer to the Swagger docs (links below) for more information.
+In this process, you might need to use the [`up_import`](../standards/rpc-api#up_import) RPC call from the [Universal Profile Extension](/install-up-browser-extension) in order to add the deployed UP to the browser extension.
 
-In this process, you might need to use the [`up_import`](../standards/rpc-api#up_import) RPC call from the [Universal Profile Extension](/install-up-browser-extension).
+<details>
+  <summary>OPTION 1: lsp6ControllerAddress & lsp3Profile</summary>
```
[the remark this answers is not in the document]
Comment (magalimorin18) [reply docs/tools/relayer-developer.md:37]: I would prefer to keep details here so the main page stays concise and if developers want more info they can open the details drop down

**Review by richtera — APPROVED** [review]

**Review by CJ42 — APPROVED** [review]

**Pull request merged**